mne/io/ctf/ctf.py

```diff
@@ -129,6 +129,7 @@ def __init__(self, directory, system_clock='truncate', preload=False,
             if sample_info['n_samp'] == 0:
                 break
             if len(fnames) == 0:
+                # XXX: Buffer size different to c version
                 info['buffer_size_sec'] = \
                     sample_info['block_size'] / info['sfreq']
```

Comment:
Sorry, I was looking at the wrong stim channel. Maybe there isn't a bug. Will check tomorrow...